apps/Sublime Text/install-64

```diff
@@ -1,6 +1,10 @@
 #!/bin/bash
-echo "Adding GPG key..."
-wget -qO- https://download.sublimetext.com/sublimehq-pub.gpg | sudo apt-key add - || error "Failed to add GPG key for sublime-text repository!"
+
+## Install dependencies
+install_packages wget gpg || error "Failed to install dependencies"
+
+status "Adding GPG key..."
+gpg --keyserver keyserver.ubuntu.com --recv-keys F57D4F59BD3DF454 || error "Failed to add GPG key for sublime-text repository!"
```

Comment:
The current script gets a gpg key from https://download.sublimetext.com/sublimehq-pub.gpg, but this proposed modification would get the key from keyserver.ubuntu.com. Is this a security risk? Also, if the imported key is written to a universal location then it seems like this would need to run with `sudo.

Reply:
Pretty sure it should work without keyserver option too. In that case it will use the keyservers of gpg maintainers maybe.

Reply:
we already have two other valid methods for replacing apt-key, I'm not a fan of obtaining the key from the ubuntu server over gettting it directly from sublimetext.

now that I'm looking for it, I can't find the issue/discussion to link to where the methods are detailed but botspot knows what I'm talking about

Reply:
found them:
https://github.com/Botspot/pi-apps/pull/1673#issuecomment-1095351120

https://github.com/Botspot/pi-apps/pull/1673#issuecomment-1106777695

Reply:
About the first method, I tried the following but it didn’t work:
```bash
sudo curl https://download.sublimetext.com/sublimehq-pub.gpg -o /etc/apt/trusted.gpg.d/sublimehq-pub.gpg
sudo apt update
```
But it didn’t work